packages/attestation-service/src/requestHandlers/get_attestation.ts

```diff
@@ -36,10 +36,47 @@ class GetAttestationRequestHandler {
     this.key = getAttestationKey(getRequest)
   }
 
-  async getAttestationRecord() {
-    return findAttestationByKey(this.key, {
+  async withAttestationAndSecurityCodeChecked(f: (attestation: AttestationModel) => any) {
+    const transaction = await sequelize!.transaction({
       logging: this.sequelizeLogger,
+      type: Transaction.TYPES.IMMEDIATE,
```

Comment:
is IMMEDIATE not the default for db transaction options?  what does having it here ensure?

Reply:
It fixes issues with sqlite described here: https://activesphere.com/blog/2018/12/24/understanding-sqlite-busy

Reply:
oh interesting, "IMMEDIATE behaviour uses a different kind of lock which allows concurrent readers, but blocks other concurrent writers". you'd think that would be accomplished just by obtaining the lock..